content/pages/info.rst

```diff
@@ -5,9 +5,10 @@ Information
 WiFi
 ----
 
-We are using the OSU public wifi system. Look for **OSU_ACCESS** (choose local
-DB), account name is **BBC14**, password is **sleepyMandrill** (this account
-will work only for the weekend).
+We are using the OSU public wifi system. If you're an OSU student, we 
+suggest using OSU Secure and signing in with your ONID. If you're not
```

Comment:
You should have this be bolded and actually using the correct SSID name which is OSU_Secure.